Source/CesiumRuntime/Public/CesiumMetadataFeatureTable.h

```diff
@@ -95,7 +95,7 @@ class CESIUMRUNTIME_API UCesiumMetadataFeatureTableBlueprintLibrary
       BlueprintCallable,
       BlueprintPure,
       Category = "Cesium|Metadata|FeatureTable")
-  static TMap<FString, FCesiumMetadataGenericValue> GetPropertiesForFeatureID(
+  static TMap<FString, FCesiumMetadataGenericValue> GetMetadataValuesForFeatureID(
```

Comment:
I discussed with @argallegos offline, and `GetPropertiesForFeatureID` name is not suitable since it returns a map between property name and MetadataGenericValue. We have another similar method name `GetMetadataValuesForFace` in `CesiumMetadataUtilityBlueprintLibrary`. So I think this change will be consistent with the Utility's method. But this may introduce breaking changes here